introduction.md

```diff
@@ -1,2 +1,3 @@
 Hello, my Name is Kevin, and I am new to the Dojo.
-My favourite CodeSchool path is JavaScript.
\ No newline at end of file
+My favourite CodeSchool path is JavaScript.
+*C#
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?